Process.h

```diff
@@ -394,7 +394,11 @@ bool Process_changePriorityBy(Process* this, Arg delta);
 
 bool Process_sendSignal(Process* this, Arg sgn);
 
-int Process_pidCompare(const void* v1, const void* v2);
+static inline int Process_pidEqual(const void* v1, const void* v2) {
+   const pid_t p1 = ((const Process*)v1)->pid;
+   const pid_t p2 = ((const Process*)v2)->pid;
+   return p1 != p2; /* return zero when equal */
+}
```

Comment:
I would recommend to name this function "pidDiffer". Right now it returns 0 if pids are equal. From a boolean point of view, that would be false.

Reply:
This function is used as an argument to `Vector_indexOf` where semantics of the return value are like `strcmp`.